portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java

```diff
@@ -53,7 +53,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     List<Comment> comments =
     results.stream()
     .filter(entity -> entity.hasProperty("message"))
-    .map(entity -> new Comment(entity.getKey().getId(), (String) entity.getProperty("message"), (long) entity.getProperty("timestamp")))
+    .map(entity -> new Comment(entity.getKey().getId(), (String) entity.getProperty("name"), (String) entity.getProperty("message"), (long) entity.getProperty("timestamp")))
```

Comment:
What do you mean by converter?

Reply:
Something that converts from class A to class B. We often use a Converter class, but even a method would be useful here.

Then this line could be something like:
.map(EntityToCommentConverter::convert)

or if you used a function
.map(entity->convertEntityToComment(entity))

Reply:
Oh I see, thanks!

Reply:
The particular Converters we use are these:
https://guava.dev/releases/19.0/api/docs/com/google/common/base/Converter.html

So in this case it would be:
```
public class EntityToCommentConverter extends Converter<Entity, Comment>{
    
    @Override
    Comment doForward(Entity entity){
        ... // code to convert a comment from an entity
    }
}
```

Reply:
Thanks!

Reply:
Decided to go with a method because to use the converter class I needed to also implement the doBackward() method which I didn't see a use for.